backlog/README.md

```diff
@@ -122,6 +119,26 @@ bulk_num,trib_id
 
 This tracking ensures consistency across multiple processing runs and helps maintain referential integrity between the source tribunal system and our system.
 
+## Court Codes
+
+The `court` field in the CSV must contain a valid court code that corresponds to a Court object defined in the system. Court codes are used to identify specific courts and tribunals.
+
+### Common Court Codes
+
+Examples of valid court codes include:
+
+- `EWHC-QBD-Admin` - Administrative Court (Queen's Bench Division)
+- `EWHC-Chancery` - Chancery Division  
+- `EWHC-Family` - Family Division
+- `UKFTT-GRC` - First-tier Tribunal (General Regulatory Chamber)
+- `UKIST` - Immigration Services Tribunal (pre-2010)
+
+### Court Code Validation
```

Comment:
This section feels more important than the examples, maybe move it up (the "this is where they're defined" is particularly important)